README.md

```diff
@@ -29,7 +29,7 @@ Given an input, PECOS identifies a small set (10-100) of relevant outputs from a
 
 ## Requirements and Installation
 
-* Python (3.7, 3.8, 3.9, 3.10)
+* Python (3.8, 3.9, 3.10)
```

Comment:
I think you probably need to add `3.11` here since PyPI wheel for `3.11` will be published.